contrib/aerospike/aerospike-client.v5/option.go

```diff
@@ -0,0 +1,65 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package aerospike
+
+import (
+	"math"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal"
+)
+
+const (
+	serviceName   = "aerospike"
+	operationName = "aerospike.command"
```

Comment:
```suggestion
	defaultService = "aerospike"
	defaultOp      = "aerospike.command"
```
Might be easier to understand.

Reply:
done.